lib/filterx/filterx-globals.c

```diff
@@ -77,23 +63,49 @@ filterx_builtin_functions_init(void)
   g_assert(filterx_builtin_function_register("int", filterx_typecast_integer));
   g_assert(filterx_builtin_function_register("double", filterx_typecast_double));
   g_assert(filterx_builtin_function_register("strptime", filterx_datetime_strptime));
+  g_assert(filterx_builtin_function_register("is_type", filterx_object_is_type_builtin));
```

Comment:
[nitpick]

`isset()` and `unset()` don't have an underscore in their names. I feel like this should be `istype()` but not sure. What do you think? @bazsi @bshifter @MrAnno